bvm/ballerina-config/src/main/java/org/ballerinalang/config/cipher/AESCipherTool.java

```diff
@@ -79,7 +79,6 @@ public AESCipherTool(String userSecret) throws AESCipherToolException {
      */
     public AESCipherTool(Path userSecretFile) throws IOException, AESCipherToolException {
         List<String> userSecret = Files.readAllLines(userSecretFile, StandardCharsets.UTF_8);
-        Files.deleteIfExists(userSecretFile);
 
```

Comment:
Removing secret file after reading has been deleted based on discussion in spec issue